apps/condo/domains/banking/utils/validate/number.utils.spec.js

```diff
@@ -0,0 +1,79 @@
+const { SPACE_SYMBOLS, SPACE_SYMBOL_LABLES } = require('@condo/domains/common/utils/string.utils')
+const { validateNumber } = require('@condo/domains/banking/utils/validate/number.utils')
+const { createValidRuNumber } = require('@condo/domains/banking/utils/testSchema/bankAccountGenerate')
+
+const SPACES = SPACE_SYMBOLS.split('')
+
+const COUNTRY_CODE_RU = 'ru'
+const VALID_BANK_ACCOUNTS = [
+    {
+        bankAccount: '40647859100000003330',
```

Comment:
It's easier to understand the code when variable names are matching appropriate fields in the BankAccount model.
```suggestion
        number: '40647859100000003330',
```